python/ctsm/test/test_sys_py_env_create.py

```diff
@@ -327,7 +334,8 @@ def test_complete_py_env_create(self):
             raise e
         env_list = get_conda_envs()
         for env_name in self.env_names:
-            assert does_env_exist(env_name, env_list)
+            if not does_env_exist(env_name, env_list):
```

Comment:
This seems like this should have worked previously. Does this result in better error handling or messaging?